src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -10,4 +10,8 @@
     Optional<CinemaHall> get(Long id);
 
     List<CinemaHall> getAll();
+
+    boolean update(CinemaHall cinemaHall);
+
+    boolean delete(Long id);
```

Comment:
Consider changing the return type of the `delete` method to `void` to better reflect the operation's outcome. Returning a `boolean` might not be necessary if the method is expected to always succeed or throw an exception otherwise.